src/utils/fs.ts

```diff
@@ -1,28 +1,19 @@
-import fs from 'fs';
+import { promises as fs } from 'fs';
 import { dirname } from './path';
 
 export * from 'fs';
 
-export const readFile = (file: string): Promise<string> =>
-	new Promise<string>((fulfil, reject) =>
-		fs.readFile(file, 'utf-8', (err, contents) => (err ? reject(err) : fulfil(contents)))
-	);
+export function readFile(file: string): Promise<string> {
+	return fs.readFile(file, 'utf8');
+}
 
-function mkdirpath(path: string) {
+async function mkdirpath(path: string): Promise<void> {
 	const dir = dirname(path);
-	fs.mkdirSync(dir, { recursive: true });
+	await fs.mkdir(dir, { recursive: true });
```

Comment:
While we are inlining things, considering this has recently been reduced to two lines and the only usage location is also a very short one, maybe inline it as well? But you can also keep it separate if you prefer.